packages/ui-components/src/components/switch-button/switch-button.tsx

```diff
@@ -58,25 +45,22 @@ export class KvSwitchButton {
 
 		return (
 			<Host>
-				<div class="switch-button-container">
-					{this.hasLabel && <span class="label">{this.label}</span>}
-					<div class="switch-button-content">
-						<div
-							class={{
-								'switch-button': true,
-								'switch-button--disabled': this.disabled,
-								'switch-button--on': this.checked,
-								[`switch-button--${this.size}`]: true
-							}}
-							part="button"
-							onClick={this.onSwitchClick}
-						>
-							<div class="icon-square" part="icon-square">
-								<kv-icon name={iconName} part="icon-svg" />
-							</div>
-						</div>
+				<slot name="left-slot" />
+				<div
+					class={{
+						'switch-button': true,
+						'switch-button--disabled': this.disabled,
+						'switch-button--on': this.checked,
+						[`switch-button--${this.size}`]: true
+					}}
+					part="button"
+					onClick={this.onSwitchClick}
+				>
+					<div class="icon-square" part="icon-square">
+						<kv-icon name={iconName} part="icon-svg" />
 					</div>
 				</div>
+				<slot name="right-slot" />
 			</Host>
```

Comment:
Should the right/left slots be suitable for changing the checkbox state?
Normally when we click on the switch label, it changes the check state.

Reply:
Yes, it's true, but this will be handled by the parent that manages these slots, not in these components.